packages.adoc

```diff
@@ -109,6 +109,34 @@ license usage. To find out more about {product-short} Archiving and Discovery,
 contact {product-short} sales.
 endif::networkeditiondoc[]
 
+* *{product-short} Chat:* Installing this package is optional. This package should be installed on at least one
+*{product-short}-Store* server. *{product-short} Chat includes a server extension with all 
+the necessary modules to run an embedded XMPP Server, and an End-User Zimlet which connects
+to the server extension and offer to the users a rich point-to-point text-chat conversations.
+{product-short} Chat is marked as GA and supported starting {product-name} 8.8.
++
+[NOTE]
+The {product-short} Chat package must be installed on the different *{product-short}-Store* server
+while having a multi-server environment.
++
+
+* *{product-short} Drive:* Installing this package is optional.This package should be installed on at least one
+*{product-short}-Store* server. *{product-short} Drive includes a server extension with all 
+the necessary modules to connect and authenticate *{product-short} Users against a ownCloud or
+NextCloud Server, and an End-User Zimlet which allow users to perform actions to the their docu-
+ments stored on ownCloud or Nextcloud.
+{product-short} Drive is marked as GA and supported starting {product-name} 8.8.
++
+[NOTE]
+The {product-short} Drive package must be installed on the different *{product-short}-Store* server
+while having a multi-server environment.
++
+
+[WARNING]
+Zimbra Drive provides only a connectivity to a ownCloud or NextCloud Server. And is the Customer
+responsibility to maintain, backup, and protect the data stored on this OC or NC Servers.
```

Comment:
s/on this OC or NC Servers/on these ownCloud or NextCloud Servers/

Reply:
done